modules/ROOT/pages/alerts.adoc

```diff
@@ -5,18 +5,16 @@ endif::[]
 
 Alerts indicate whether a resource (such as a Mule app) is behaving as expected or exceeding a defined threshold.
 
-There are several types of alerts. From the Alerts page in Anypoint Monitoring, you can link out to alerts for API and Runtime alerts. If your subscription permits, you can also view, enable, and disable alerts that are configured for graphs in custom dashboards.
+There are several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:alerts-config.adoc#basic-alerts[basic alerts] or link out to alerts for API Manager and Runtime Manager alerts.
```

Comment:
Try to avoid "There is" or "There are" as the "subject" of a sentence. 
Would this work?
```suggestion
You can implement any of several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:alerts-config.adoc#basic-alerts[basic alerts], or you can link to alerts for API Manager and Runtime Manager alerts.
```